doc/source/conf.py

```diff
@@ -183,6 +183,12 @@
     message=r'There is no current event loop',
     category=DeprecationWarning,
 )
+# warnings in sparse for isspmatrix and isspmatrix_fmt
+warnings.filterwarnings(
```

Comment:
can you add this here instead so we are not doing the same thing in multiple places
https://github.com/scipy/scipy/blob/763a25e17379c43529bef5e94f0a7d73ac07969a/doc/source/conf.py#L316-L326